app/helpers/flash_helper.rb

```diff
@@ -1,6 +1,6 @@
 module FlashHelper
   # Flash type -> Bootstrap alert class
   def flash_mapping
-    { notice: :info, error: :danger, success: :success }
+    { notice: :info, error: :danger, authorization_error: :danger, success: :success }
```

Comment:
Wouldn't it  make sense to have a `:warning` here, that could be used by other places here instead of a specific `:authorization_error`?